module-services/service-desktop/WorkerDesktop.cpp

```diff
@@ -97,6 +97,18 @@ bool WorkerDesktop::reinit(const std::filesystem::path &path)
     return true;
 }
 
+ void WorkerDesktop::reset()
+ {
+    usbStatus = bsp::USBDeviceStatus::Disconnected;
+    bsp::usbDeinit();
+
+    bsp::usbInitParams initParams = {receiveQueue, irqQueue, serialNumber.c_str()};
+    initialized = bsp::usbInit(initParams) >= 0;
+    ownerService->bus.sendMulticast(std::make_shared<sdesktop::usb::USBConnected>(),
```

Comment:
so we're sending a USB initialized message even when initialization failed?

Reply:
oh no, we should not do that. Will be fixed